### ISSUE TITLE
Make OneDrive optional

### ISSUE THREAD

**Issue by ptrkhh** [bmrf/tron#160]
OneDrive removal should be made optional, both using a command line flag, and in tron_settings.bat

**Comment by vocatus** [comment]
OneDrive is only removed if it's not in use; are you saying add a switch to completely ignore it regardless whether its in use?

**Comment by ptrkhh** [comment]
I would say so. The reason I noticed this is because I used this script right after installation on my laptop, before I had the chance to configure my online accounts. This would also be useful for those who want to prepare images (sysprep) for a laptop/prebuilt/tablet/etc.

Also I tried it on my personal PC with OneDrive still active, and my personal OneDrive folder is removed from the File Explorer's Navigation Pane (the tree on the left), although the syncing still operate normally. Weirdly enough, my work OneDrive stays, only my personal OneDrive is gone from the navigation pane.

Either way, I think adding an option is always a good idea. Its a good peace of mind IMO to have  OneDrive intact without relying on the script's detection method.
